src/app/public/src/modules/action-buttons/action-buttons.component.scss

```diff
@@ -9,6 +9,13 @@
   display: inline-flex;
 }
 
+.stache-action-buttons-search-container {
+  position: relative;
+  display: flex;
+  min-height: 49px;
+  margin-bottom: 10px;
+}
```

Comment:
Some strange spacing because of the `stache-row` on the action buttons?   Do we want to mimic the `-15px` margins on the left and right that the `stache-row` implements?